practices/java-web/HelloWorldServletMvcWebApplication/src/main/java/com/dowa/java/web/HelloWorldMvcServlet.java

```diff
@@ -20,11 +21,28 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
     @Override
     protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
 
-        RequestDispatcher requestDispatcher = req.getRequestDispatcher("/Hello.jsp");
+        RequestDispatcher requestDispatcher1 = req.getRequestDispatcher("/Hello.jsp");
+        RequestDispatcher requestDispatcher2 = req.getRequestDispatcher("/Hello2.jsp");
```

Comment:
this violates DRY principle

how about :
String url = "hello.jsp";
if (names.contains(name)) {
url = "a";
}
else {
url = "b"
}
RequestDispatcher rd = req.getRequestDispatcher(url)
...